Moved the customization guide links up for better visibility (#6362)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/android/content_cards/integration.md

> This reference article covers the Content Card integration and the different data models and card-specific properties available for your Android or FireOS application.

{% alert note %}

> Once you understand the basics of Content Cards, see the [Content Card Customization Guide]({{site.baseurl}}/developer_guide/customization_guides/content_cards) to get started with implementation and customization.

Reviewer: ```suggestion
When you're ready to get started with implementation and customization, see the [Content Card Customization Guide]({{site.baseurl}}/developer_guide/customization_guides/content_cards).
```

```suggestion
When you're ready to implement and customize Content Cards, see the [Content Card Customization Guide]({{site.baseurl}}/developer_guide/customization_guides/content_cards).
```

Small suggestion per the style guide:
Use to refer to performing an action a single time. Don’t use “once” in place of “after”.
